src/nashpy/algorithms/support_enumeration.py

```diff
@@ -73,7 +77,9 @@ def solve_indifference(A, rows=None, columns=None):
         return False
 
 
```

Comment:
It looks like this line:

```
except np.linalg.linalg.LinAlgError:
```

can be changed to:

```
except np.linalg.LinAlgError:
```

Once you do that it looks like you will also need to remove some unused imports: `Iterable, List, Optional`